content/en/docs/reference/glossary/cidr.md

```diff
@@ -0,0 +1,18 @@
+---
+title: CIDR
+id: cidr
+date: 2019-11-12
+full_link: 
+short_description: >
+  CIDR is a notation for describing blocks of IP addresses and is used heavily in various networking configurations.
```

Comment:
```suggestion
  A notation for describing blocks of IP addresses
```

Reply:
This could be a glossary tooltip. I think these work well when succinct; someone who wants to learn more can click.